src/ngx-gallery-preview.component.ts

```diff
@@ -432,15 +444,18 @@ export class NgxGalleryPreviewComponent implements OnInit, OnChanges {
         this.resetPosition();
 
         this.src = this.getSafeUrl(<string>this.images[this.index]);
+        this.type = this.getFileType(<string>this.images[this.index]);
         this.srcIndex = this.index;
         this.description = this.descriptions[this.index];
         this.changeDetectorRef.markForCheck();
 
         setTimeout(() => {
-            if (this.isLoaded(this.previewImage.nativeElement)) {
+            if (this.isLoaded(this.previewImage.nativeElement) || this.type == 'video') {
                 this.loading = false;
                 this.startAutoPlay();
                 this.changeDetectorRef.markForCheck();
+            } else if (this.type == 'video') {
+                
```

Comment:
I feel that videos could require something different than what is inside the above block. I kept this condition, with an empty block, it to highlight that. 

So, this condition `else if (this.type == 'video')` should either be removed along with its  empty block. Or, the condition ` || this.type == 'video'` should be removed and the empty block should be filled of what is required.